composer.json

```diff
@@ -18,7 +18,7 @@
         "zendframework/zend-servicemanager": "dev-develop as 2.6.0",
```

Comment:
No; it'll be `^2.7 || ^3.0`; however, any such changes need to wait; I'm doing a few migrations now, and seeing exactly what steps are needed, and what changes to make. For this particular component, and for zend-modulemanager, I'll actually be able to undo a lot of the changes we created previously, as they're accommodated by 2.7 and 3.0.